### ISSUE TITLE
Missing ErrorCode constant for "card_decline_rate_limit_exceeded" in error.go

### ISSUE THREAD

**Issue by jinjlu** [stripe/stripe-go#1077]
go version: go1.13.4 darwin/amd64
stripe library: Stripe/v1 GoBindings/67.4.1

The error code is listed in the [doc](https://stripe.com/docs/error-codes#card-decline-rate-limit-exceeded) but the constant is not presented in error.go in latest change or the version I used. 
a sample response:
```
POST /v1/charges
{
  "error": {
    "code": "card_decline_rate_limit_exceeded",
    "doc_url": "https://stripe.com/docs/error-codes/card-decline-rate-limit-exceeded",
    "message": "You have exceeded the maximum number of declines on this card in the last 24 hour period. Please contact us via https://support.stripe.com/contact if you need further assistance.",
    "type": "invalid_request_error"
  }
}
```

**Comment by remi-stripe** [comment]
Thanks for the report. I opened a PR in https://github.com/stripe/stripe-go/pull/1078 and it will be fixed in the next version

**Comment by jinjlu** [comment]
Awesome! Thanks for fixing it!

**Comment by remi-stripe** [comment]
This is now fixed in 71.3.0, thanks again for the report!
